### DISCUSSION TITLE
Creation of Remote Homology dataset

### DISCUSSION THREAD

**Discussion by Alexzhuan** [facebookresearch/esm#217] [Q&A]
Hi,

I really appreciate the significant work!

The detection of remote homology (refer to [1]) is a reasonable task to evaluate the capacity of PLM (e.g., ESM-1b). In our recent work, we are also trying to use this downstream task to investigate the effectiveness of our model.  

Following the method in the *Supplementary Information* for [1], I construct a remote homology dataset using SCOP 2.07e provided by the ASTRAL compendium, consisting of 1219 unique folds, which is different from what is mentioned in the *Supplementary Information*
>The dataset consists of 217 unique folds and 366 unique superfamilies

I also notice the number of folds in the Statistics of [SCOPe website](https://scop.berkeley.edu/statistics/ver=2.07) is 1232. Whether there are any necessary steps in the construction of the dataset that I missed? (e.g., excluding some folds)

Thanks!

[1] Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences.

**Comment by jasperhyp** [comment]
I'm also wondering if the processed evaluation datasets can be released.
